src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner sc;
+    private String input = "";
+
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm duke.Duke\n" + "What can I do for you?");
+    }
+
+    public void goodbye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public void addTaskToList(Task task, int size) {
+        String taskToString = task.toString();
+        System.out.println("Got it. I've added this task: \n" + taskToString
+                + "\nNow you have " + size + " tasks in the list.");
+    }
+
+    public void removeTaskFromList(Task task, int size) {
+        String taskToString = task.toString();
+        System.out.println("Noted. I've removed this task: \n" +
+                taskToString
+                + "\nNow you have " + size + " tasks in the list.");
+    }
```

Comment:
It's possible to just use `task` here as the conversion to String type is done implicitly (like how you don't have to convert `size` to a String).
```suggestion
    public void addTaskToList(Task task, int size) {
        System.out.println("Got it. I've added this task: \n" + task
                + "\nNow you have " + size + " tasks in the list.");
    }

    public void removeTaskFromList(Task task, int size) {
        System.out.println("Noted. I've removed this task: \n" +
                task
                + "\nNow you have " + size + " tasks in the list.");
    }
```